sigar/src/test/java/io/crate/monitor/SigarExtendedNodeInfoTest.java

```diff
@@ -62,7 +64,11 @@ public void cleanup() throws Exception {
     public void testNetworkStats() throws Exception {
         ExtendedNetworkStats stats = extendedNodeInfo.networkStats();
         assertThat(stats.timestamp(), greaterThan(0L));
-        assertThat(stats.tcp().activeOpens(), greaterThan(0L));
+        if (isNewMacOSX()) {
```

Comment:
@seut according to: http://cpansearch.perl.org/src/DOUGM/hyperic-sigar-1.6.3-src/docs/javadoc/org/hyperic/sigar/Tcp.html#getActiveOpens() it executes `cat /proc/net/snmp` which is missing from new MacOSX high Sierra, but maybe that's not the issue. When you have time please try it locally on your machine.

Reply:
works for me on 10.12